PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBBidRequest.m

```diff
@@ -68,6 +68,58 @@ - (nonnull PBMJsonDictionary *)toJsonDictionary {
     ext[@"prebid"] = [[self.extPrebid toJsonDictionary] nullIfEmpty];
     ret[@"ext"] = [[ext pbmCopyWithoutEmptyVals] nullIfEmpty];
     
+    //remove "protected" fields from ortbObject then do a merge but merge ret into the ortbObject (addEntriesFromDictionary)
+    NSMutableDictionary *o = [self.arbitraryJsonConfig mutableCopy];
+    if (o[@"regs"]) {
+        o[@"regs"] = nil;
+    }
+    if (o[@"device"]) {
+        o[@"device"] = nil;
+    }
+    if (o[@"geo"]) {
+        o[@"geo"] = nil;
+    }
+    //merge with config from API/JSON
+    ret = [self mergeDictionaries: ret joiningArgument2: o joiningArgument3: false];
+    
+    NSMutableDictionary *o2 = [self.ortbObject mutableCopy];
+    
+    if (o2[@"regs"]) {
+        o2[@"regs"] = nil;
+    }
+    if (o2[@"device"]) {
+        o2[@"device"] = nil;
+    }
+    if (o2[@"geo"]) {
+        o[@"geo"] = nil;
+    }
+    //merge with ortbConfig from SDK
+    ret = [self mergeDictionaries: ret joiningArgument2: o2 joiningArgument3: true];
+    
+    ret = [ret pbmCopyWithoutEmptyVals];
+    
+    return ret;
+}
+
+- (nonnull PBMMutableJsonDictionary *)mergeDictionaries:(NSMutableDictionary*)dictionary1 joiningArgument2:(NSMutableDictionary*)dictionary2
```

Comment:
In general, we can merge dictionaries. This path is faster to implement. However, it makes the solution restricted to such things as 
- RTB protocol validation (proper structure, objects, and types)
- Warnings for objects prohibited from merging. 
- Upgrading protocol version. See the item 1 - the validation issue. 

It is not critical, but I would still merge serialized objects.  It is a more reliable approach from the support perspective.